app/src/components/Register.tsx

```diff
@@ -107,13 +105,13 @@ class Register extends React.Component<RegisterProps, RegisterState> {
       .then((total: number) => {
         const artists = [];
 
-        let id = 0;
-        while (id < total) {
-          id++;
-
+        let i = 0;
+        while (i < total) {
```

Comment:
Would a for loop be more readable here `for (let i = 1; i <= total; i++) { ... }`?